vars/buildPlugin.groovy

```diff
@@ -95,6 +116,26 @@ def call(Map params = [:]) {
                             }
 
                             junit testReports // TODO do this in a finally-block so we capture all test results even if one branch aborts early
+                            if (isMaven && archiveFindbugs) {
+                                def fp = [pattern: params?.findbugs?.pattern ?: '**/findbugsXml.xml']
```

Comment:
I think you can select a more specific pattern match. Certainly results will be inside `**/target/` somewhere.